gadget/gadget.go

```diff
@@ -532,6 +620,9 @@ type Size uint64
 const (
 	SizeMiB = Size(2 << 20)
 	SizeGiB = Size(2 << 30)
+
+	SizeMBR          = Size(446)
```

Comment:
Could you please add a comment for this. Even something as simple as `SizeMBR is the maximum size of the master boot record`.

Reply:
Well, the truth it it's not the actual size of MBR (as in Master Boot Record), which has 512 bytes (1 sector), but more of the space available for bootstrap code in a classic MBR setup, which is 446. It also (incorrectly?) is referred to as `'mbr'` throughout ubuntu-image parser code and gadget.yaml.

I suppose, a description like `SizeMBR is the maximum size of a structure of 'mbr' role` is closest to reality/use.

Reply:
Added a note.